### ISSUE TITLE
Customer account navigation block fails (problem with Luma copy)

### ISSUE THREAD

**Issue by cvpportraits** [magento/magento2#7631]
fresh install of **Magento 2.1.2**, everything is working great, except when customer tries to create new account... the following error is displayed (with store in development mode so errors can print instead of the friendly error message)

**_Steps to produce error:_**
this error comes up both when creating a new customer account or, trying to navigate to: /customer/account (as the link in the welcome email provides)

I have cleared all caches, re-compiled, etc.

looking in the folder structure, vendor/magento/module-customer/block/account/Navigation.php does NOT exist.

and in app/code there is no Magento folder at all, everything seems to be in the vendor/magento folder.

**_NOTE_**
the customer's account IS created, but this error comes because of an apparent block problem with account navigation?

I have also tried, Removing /var folder and tried again, same result.

_note, I tried 3 separate installs of Magento and no matter what, all 3 separate instances all do this exact same thing._ 

Any suggestions to fix would be greatly appreciated.

```
2 exception(s):
Exception #0 (Magento\Framework\Exception\LocalizedException): Invalid block type: Magento\Customer\Block\Account\Navigation
Exception #1 (ReflectionException): Class Magento\Customer\Block\Account\Navigation does not exist

Exception #0 (Magento\Framework\Exception\LocalizedException): Invalid block type: Magento\Customer\Block\Account\Navigation
#0 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(237): Magento\Framework\View\Layout\Generator\Block->getBlockInstance('Magento\\Custome...', Array)
#1 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(216): Magento\Framework\View\Layout\Generator\Block->createBlock('Magento\\Custome...', 'customer_accoun...', Array)
#2 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(124): Magento\Framework\View\Layout\Generator\Block->generateBlock(Object(Magento\Framework\View\Layout\ScheduledStructure), Object(Magento\Framework\View\Layout\Data\Structure), 'customer_accoun...')
#3 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/GeneratorPool.php(86): Magento\Framework\View\Layout\Generator\Block->process(Object(Magento\Framework\View\Layout\Reader\Context), Object(Magento\Framework\View\Layout\Generator\Context))
#4 /home/cvpedu/public_html/vendor/magento/framework/View/Layout.php(327): Magento\Framework\View\Layout\GeneratorPool->process(Object(Magento\Framework\View\Layout\Reader\Context), Object(Magento\Framework\View\Layout\Generator\Context))
#5 /home/cvpedu/public_html/var/generation/Magento/Framework/View/Layout/Interceptor.php(89): Magento\Framework\View\Layout->generateElements()
#6 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Builder.php(129): Magento\Framework\View\Layout\Interceptor->generateElements()
#7 /home/cvpedu/public_html/vendor/magento/framework/View/Page/Builder.php(55): Magento\Framework\View\Layout\Builder->generateLayoutBlocks()
#8 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Builder.php(65): Magento\Framework\View\Page\Builder->generateLayoutBlocks()
#9 /home/cvpedu/public_html/vendor/magento/framework/View/Page/Config.php(197): Magento\Framework\View\Layout\Builder->build()
#10 /home/cvpedu/public_html/vendor/magento/framework/View/Page/Config.php(217): Magento\Framework\View\Page\Config->build()
#11 /home/cvpedu/public_html/vendor/magento/module-customer/Controller/Account/Index.php(40): Magento\Framework\View\Page\Config->getTitle()
#12 /home/cvpedu/public_html/var/generation/Magento/Customer/Controller/Account/Index/Interceptor.php(24): Magento\Customer\Controller\Account\Index->execute()
#13 /home/cvpedu/public_html/vendor/magento/framework/App/Action/Action.php(102): Magento\Customer\Controller\Account\Index\Interceptor->execute()
#14 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(74): Magento\Framework\App\Action\Action->dispatch(Object(Magento\Framework\App\Request\Http))
#15 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(70): Magento\Customer\Controller\Account\Index\Interceptor->___callParent('dispatch', Array)
#16 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'storeCheck')
#17 /home/cvpedu/public_html/vendor/magento/module-store/App/Action/Plugin/StoreCheck.php(44): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#18 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Store\App\Action\Plugin\StoreCheck->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#19 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'contextPlugin')
#20 /home/cvpedu/public_html/vendor/magento/module-store/App/Action/Plugin/Context.php(106): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#21 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Store\App\Action\Plugin\Context->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#22 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'customer-app-ac...')
#23 /home/cvpedu/public_html/vendor/magento/module-customer/Model/App/Action/ContextPlugin.php(61): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#24 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Customer\Model\App\Action\ContextPlugin->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#25 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'weee-app-action...')
#26 /home/cvpedu/public_html/vendor/magento/module-weee/Model/App/Action/ContextPlugin.php(112): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#27 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Weee\Model\App\Action\ContextPlugin->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#28 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'tax-app-action-...')
#29 /home/cvpedu/public_html/vendor/magento/module-tax/Model/App/Action/ContextPlugin.php(91): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#30 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Tax\Model\App\Action\ContextPlugin->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#31 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(138): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'customer_accoun...')
#32 /home/cvpedu/public_html/vendor/magento/module-customer/Controller/Plugin/Account.php(60): Magento\Customer\Controller\Account\Index\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#33 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(142): Magento\Customer\Controller\Plugin\Account->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#34 /home/cvpedu/public_html/var/generation/Magento/Customer/Controller/Account/Index/Interceptor.php(39): Magento\Customer\Controller\Account\Index\Interceptor->___callPlugins('dispatch', Array, Array)
#35 /home/cvpedu/public_html/vendor/magento/framework/App/FrontController.php(55): Magento\Customer\Controller\Account\Index\Interceptor->dispatch(Object(Magento\Framework\App\Request\Http))
#36 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(74): Magento\Framework\App\FrontController->dispatch(Object(Magento\Framework\App\Request\Http))
#37 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(70): Magento\Framework\App\FrontController\Interceptor->___callParent('dispatch', Array)
#38 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'requestPreproce...')
#39 /home/cvpedu/public_html/vendor/magento/module-store/App/FrontController/Plugin/RequestPreprocessor.php(94): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#40 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Store\App\FrontController\Plugin\RequestPreprocessor->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#41 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'install')
#42 /home/cvpedu/public_html/vendor/magento/framework/Module/Plugin/DbStatusValidator.php(69): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#43 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Framework\Module\Plugin\DbStatusValidator->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#44 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'front-controlle...')
#45 /home/cvpedu/public_html/vendor/magento/module-page-cache/Model/App/FrontController/BuiltinPlugin.php(73): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#46 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\PageCache\Model\App\FrontController\BuiltinPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#47 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(138): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'front-controlle...')
#48 /home/cvpedu/public_html/vendor/magento/module-page-cache/Model/App/FrontController/VarnishPlugin.php(55): Magento\Framework\App\FrontController\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#49 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(142): Magento\PageCache\Model\App\FrontController\VarnishPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#50 /home/cvpedu/public_html/var/generation/Magento/Framework/App/FrontController/Interceptor.php(26): Magento\Framework\App\FrontController\Interceptor->___callPlugins('dispatch', Array, Array)
#51 /home/cvpedu/public_html/vendor/magento/framework/App/Http.php(135): Magento\Framework\App\FrontController\Interceptor->dispatch(Object(Magento\Framework\App\Request\Http))
#52 /home/cvpedu/public_html/vendor/magento/framework/App/Bootstrap.php(258): Magento\Framework\App\Http->launch()
#53 /home/cvpedu/public_html/index.php(39): Magento\Framework\App\Bootstrap->run(Object(Magento\Framework\App\Http))
#54 {main}

Exception #1 (ReflectionException): Class Magento\Customer\Block\Account\Navigation does not exist
#0 /home/cvpedu/public_html/vendor/magento/framework/Code/Reader/ClassReader.php(19): ReflectionClass->__construct('Magento\\Custome...')
#1 /home/cvpedu/public_html/vendor/magento/framework/ObjectManager/Definition/Runtime.php(44): Magento\Framework\Code\Reader\ClassReader->getConstructor('Magento\\Custome...')
#2 /home/cvpedu/public_html/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php(71): Magento\Framework\ObjectManager\Definition\Runtime->getParameters('Magento\\Custome...')
#3 /home/cvpedu/public_html/vendor/magento/framework/ObjectManager/ObjectManager.php(57): Magento\Framework\ObjectManager\Factory\Dynamic\Developer->create('Magento\\Custome...', Array)
#4 /home/cvpedu/public_html/vendor/magento/framework/View/Element/BlockFactory.php(45): Magento\Framework\ObjectManager\ObjectManager->create('Magento\\Custome...', Array)
#5 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(257): Magento\Framework\View\Element\BlockFactory->createBlock('Magento\\Custome...', Array)
#6 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(237): Magento\Framework\View\Layout\Generator\Block->getBlockInstance('Magento\\Custome...', Array)
#7 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(216): Magento\Framework\View\Layout\Generator\Block->createBlock('Magento\\Custome...', 'customer_accoun...', Array)
#8 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Generator/Block.php(124): Magento\Framework\View\Layout\Generator\Block->generateBlock(Object(Magento\Framework\View\Layout\ScheduledStructure), Object(Magento\Framework\View\Layout\Data\Structure), 'customer_accoun...')
#9 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/GeneratorPool.php(86): Magento\Framework\View\Layout\Generator\Block->process(Object(Magento\Framework\View\Layout\Reader\Context), Object(Magento\Framework\View\Layout\Generator\Context))
#10 /home/cvpedu/public_html/vendor/magento/framework/View/Layout.php(327): Magento\Framework\View\Layout\GeneratorPool->process(Object(Magento\Framework\View\Layout\Reader\Context), Object(Magento\Framework\View\Layout\Generator\Context))
#11 /home/cvpedu/public_html/var/generation/Magento/Framework/View/Layout/Interceptor.php(89): Magento\Framework\View\Layout->generateElements()
#12 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Builder.php(129): Magento\Framework\View\Layout\Interceptor->generateElements()
#13 /home/cvpedu/public_html/vendor/magento/framework/View/Page/Builder.php(55): Magento\Framework\View\Layout\Builder->generateLayoutBlocks()
#14 /home/cvpedu/public_html/vendor/magento/framework/View/Layout/Builder.php(65): Magento\Framework\View\Page\Builder->generateLayoutBlocks()
#15 /home/cvpedu/public_html/vendor/magento/framework/View/Page/Config.php(197): Magento\Framework\View\Layout\Builder->build()
#16 /home/cvpedu/public_html/vendor/magento/framework/View/Page/Config.php(217): Magento\Framework\View\Page\Config->build()
#17 /home/cvpedu/public_html/vendor/magento/module-customer/Controller/Account/Index.php(40): Magento\Framework\View\Page\Config->getTitle()
#18 /home/cvpedu/public_html/var/generation/Magento/Customer/Controller/Account/Index/Interceptor.php(24): Magento\Customer\Controller\Account\Index->execute()
#19 /home/cvpedu/public_html/vendor/magento/framework/App/Action/Action.php(102): Magento\Customer\Controller\Account\Index\Interceptor->execute()
#20 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(74): Magento\Framework\App\Action\Action->dispatch(Object(Magento\Framework\App\Request\Http))
#21 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(70): Magento\Customer\Controller\Account\Index\Interceptor->___callParent('dispatch', Array)
#22 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'storeCheck')
#23 /home/cvpedu/public_html/vendor/magento/module-store/App/Action/Plugin/StoreCheck.php(44): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#24 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Store\App\Action\Plugin\StoreCheck->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#25 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'contextPlugin')
#26 /home/cvpedu/public_html/vendor/magento/module-store/App/Action/Plugin/Context.php(106): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#27 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Store\App\Action\Plugin\Context->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#28 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'customer-app-ac...')
#29 /home/cvpedu/public_html/vendor/magento/module-customer/Model/App/Action/ContextPlugin.php(61): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#30 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Customer\Model\App\Action\ContextPlugin->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#31 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'weee-app-action...')
#32 /home/cvpedu/public_html/vendor/magento/module-weee/Model/App/Action/ContextPlugin.php(112): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#33 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Weee\Model\App\Action\ContextPlugin->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#34 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'tax-app-action-...')
#35 /home/cvpedu/public_html/vendor/magento/module-tax/Model/App/Action/ContextPlugin.php(91): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#36 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Tax\Model\App\Action\ContextPlugin->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#37 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(138): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Custome...', 'dispatch', Object(Magento\Customer\Controller\Account\Index\Interceptor), Array, 'customer_accoun...')
#38 /home/cvpedu/public_html/vendor/magento/module-customer/Controller/Plugin/Account.php(60): Magento\Customer\Controller\Account\Index\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#39 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(142): Magento\Customer\Controller\Plugin\Account->aroundDispatch(Object(Magento\Customer\Controller\Account\Index\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#40 /home/cvpedu/public_html/var/generation/Magento/Customer/Controller/Account/Index/Interceptor.php(39): Magento\Customer\Controller\Account\Index\Interceptor->___callPlugins('dispatch', Array, Array)
#41 /home/cvpedu/public_html/vendor/magento/framework/App/FrontController.php(55): Magento\Customer\Controller\Account\Index\Interceptor->dispatch(Object(Magento\Framework\App\Request\Http))
#42 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(74): Magento\Framework\App\FrontController->dispatch(Object(Magento\Framework\App\Request\Http))
#43 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(70): Magento\Framework\App\FrontController\Interceptor->___callParent('dispatch', Array)
#44 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'requestPreproce...')
#45 /home/cvpedu/public_html/vendor/magento/module-store/App/FrontController/Plugin/RequestPreprocessor.php(94): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#46 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Store\App\FrontController\Plugin\RequestPreprocessor->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#47 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'install')
#48 /home/cvpedu/public_html/vendor/magento/framework/Module/Plugin/DbStatusValidator.php(69): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#49 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\Framework\Module\Plugin\DbStatusValidator->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#50 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(63): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'front-controlle...')
#51 /home/cvpedu/public_html/vendor/magento/module-page-cache/Model/App/FrontController/BuiltinPlugin.php(73): Magento\Framework\Interception\Chain\Chain->Magento\Framework\Interception\Chain\{closure}(Object(Magento\Framework\App\Request\Http))
#52 /home/cvpedu/public_html/vendor/magento/framework/Interception/Chain/Chain.php(67): Magento\PageCache\Model\App\FrontController\BuiltinPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#53 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(138): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'front-controlle...')
#54 /home/cvpedu/public_html/vendor/magento/module-page-cache/Model/App/FrontController/VarnishPlugin.php(55): Magento\Framework\App\FrontController\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#55 /home/cvpedu/public_html/vendor/magento/framework/Interception/Interceptor.php(142): Magento\PageCache\Model\App\FrontController\VarnishPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#56 /home/cvpedu/public_html/var/generation/Magento/Framework/App/FrontController/Interceptor.php(26): Magento\Framework\App\FrontController\Interceptor->___callPlugins('dispatch', Array, Array)
#57 /home/cvpedu/public_html/vendor/magento/framework/App/Http.php(135): Magento\Framework\App\FrontController\Interceptor->dispatch(Object(Magento\Framework\App\Request\Http))
#58 /home/cvpedu/public_html/vendor/magento/framework/App/Bootstrap.php(258): Magento\Framework\App\Http->launch()
#59 /home/cvpedu/public_html/index.php(39): Magento\Framework\App\Bootstrap->run(Object(Magento\Framework\App\Http))
#60 {main}
```

**Comment by cvpportraits** [comment]
so, would rolling back a version fix this? I'm guessing 2.1 just isn't ready for public use yet? (every instance I install of it, has this issue... diff domains, diff servers... does anyone know of any resources besides here and stackexchange for help on this matter?

**Comment by cvpportraits** [comment]
so I have reinstalled Magento, And narrowed the issue to a new theme which is a copy of Luma per this article's instructions.... the error only happens when using this. - is there any way of using luma as a base and not having this problem, here's the article I followed: 

dckap.com/blog/creating-magento-2-theme
